code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -147,6 +146,11 @@ All foods are distributed among various categories. Use common sense.
 				var/fraction = min(bitesize / reagents.total_volume, 1)
 				reagents.trans_to(M, bitesize, transfered_by = user, method = INGEST)
 				bitecount++
+				if(iscarbon(M))
+					var/mob/living/carbon/C = M
+					var/obj/item/bodypart/head/hed = C.get_bodypart(BODY_ZONE_HEAD)
+					if(hed)
+						hed.mouthful += bitesize
```

Comment:
please don't add more kaka code to food code while it's still being moved to the component :(

Reply:
(or atleast add it to the component too)